### PR TITLE
Endpoints for getting/setting dao_fee

### PR DESCRIPTION
**Pull request by asoltys**

### CODE REVIEW

File: rita/src/rita_common/dashboard/dao.rs [hunk 1]
```diff
@@ -51,3 +53,23 @@ pub fn remove_from_dao_list(path: Path<(Address)>) -> Result<Json<()>, Error> {
...
+
+pub fn get_dao_fee(_req: HttpRequest) -> Result<HttpResponse, Error> {
+    debug!("/dao_fee GET hit");
+    let mut ret = HashMap::new();
+    ret.insert("dao_fee", SETTING.get_dao().dao_fee.to_string());
+
+    Ok(HttpResponse::Ok().json(ret))
+}
+
+pub fn set_dao_fee(path: Path<u32>) -> Result<Json<()>, Error> {
```
Comment (jkilpatr) [inline rita/src/rita_common/dashboard/dao.rs]: you should use at least a u128 here, u32 will easily be overflowed.

File: rita/src/rita_common/oracle/mod.rs [hunk 2]
```diff
@@ -306,8 +306,12 @@ fn update_our_price() {
                                                 new_prices.fee_multiplier;
                                             drop(payment);
 
-                                            let mut dao = SETTING.get_dao_mut();
-                                            dao.dao_fee = Uint256::from(new_prices.dao_fee);
+                                            let new_dao_fee = Uint256::from(new_prices.dao_fee);
+                                            let current_dao_fee = Uint256::from(SETTING.get_dao().dao_fee.clone());
```
Comment (jkilpatr) [inline rita/src/rita_common/oracle/mod.rs]: this unint256 conversion isn't needed, it's already stored that way

File: rita/src/rita_common/tunnel_manager/mod.rs [hunk 3]
```diff
@@ -281,9 +282,8 @@ impl Handler<PortCallback> for TunnelManager {
 }
 
 pub fn make_babel_stream() -> Result<TcpStream, Error> {
-    let stream = TcpStream::connect::<SocketAddr>(
-        format!("[::1]:{}", SETTING.get_network().babel_port).parse()?,
-    )?;
+    let stream = open_babel_stream(SETTING.get_network().babel_port)?;
```
Comment (jkilpatr) [inline rita/src/rita_common/tunnel_manager/mod.rs]: a lot of these babel changes shouldn't be here, did you pull in my recent babel pull yourself somehow?

**Comment by jkilpatr** [comment]
you still need to rustup update then cargo +stable rust fmt

**Comment by asoltys** [comment]
@ttk2 this should be good now

**Pull request merged**